index.html

```diff
@@ -6,7 +6,318 @@
+  </button>
+  <button class="slider-button slider-next" type="button">
+    <span class="visually-hidden">Следующее фото</span>
+  </button>
+  <img src="#" alt="пломбир с клубничным джемом " class="slider-image">
+  <img src="#" alt="Карамельный пломбир " class="slider-image">
+      <ul class="social-list">
+        <li class="social-item">
+          <a class="button" href="https://www.youtube.com/user/htmlacademyru">
+            <span class="visually-hidden">Юттуб</span>
+          </a>
+        </li>
+        <li class="social-item">
+          <a class="button" href="https://vk.com/htmlacademy">
+            <span class="visually-hidden">ВКонтакте</span>
+          </a>
+        </li>
+        <li class="social-item">
+          <a class="button" href="https://t.me/htmlacademy">
+            <span class="visually-hidden">Телеграм</span>
+          </a>
+        </li>
+      </ul>
+</section>
+<section class="three-demonstration-icecream">
+  <div class="collector">
+    <h2 class="slider-title">Карамельный пломбир с маршмеллоу</h2>
+    <p class="icecream-description">Необычный сладкий десерт с карамельным топпингом и кусочками зефира завоюет сердца сладкоежек всех возрастов.</p>
+    <a class="button-link" href="">Заказать</a>
+  </div>
+  <ol class="slider-pagination">
+    <li class="slider-pagination-item"><button class="slider-pagination-button " type="button">1 фото</button></li>
+    <li class="slider-pagination-item"><button class="slider-pagination-button "
+        type="button">2 фото</button></li>
+    <li class="slider-pagination-item"><button class="slider-pagination-button slider-pagination-button-current" type="button">3 фото</button></li>
+  </ol>
+  <img src="#" alt="Карамельный пломбир" class="slider-image ">
+  <button class="slider-button slider-prev" type="button">
+    <span class="visually-hidden">Предыдущее фото</span>
+  </button>
+  <button class="slider-button slider-next" type="button">
+    <span class="visually-hidden">Следующее фото</span>
+  </button>
+  <img src="#" alt="пломбир с клубничным джемом " class="slider-image">
+  <img src="#" alt="Сливочное мороженое " class="slider-image">
+      <ul class="social-list">
+        <li class="social-item">
+          <a class="button" href="https://www.youtube.com/user/htmlacademyru">
+            <span class="visually-hidden">Юттуб</span>
+          </a>
+        </li>
+        <li class="social-item">
+          <a class="button" href="https://vk.com/htmlacademy">
+            <span class="visually-hidden">ВКонтакте</span>
+          </a>
+        </li>
+        <li class="social-item">
+          <a class="button" href="https://t.me/htmlacademy">
+            <span class="visually-hidden">Телеграм</span>
+          </a>
+        </li>
+      </ul>
+</section>
+<section class="present">
+  <h2 class="visually-hidden ">Подарки</h2>
+  <p class="title-description">Заказывайте мороженое и получайте подарки!</p>
```

Comment:
У блока ведь уже есть заголовок.

```suggestion
  <h2 class="title-description">Заказывайте мороженое и получайте подарки!</h2>
```